### PR TITLE
4886-Epicea-cant-multi-select-sessions

### LINKED ISSUE #4886: Epicea: can't multi-select sessions
Extracted from issue #4496

### PR DESCRIPTION
**Pull request by tinchodias**
Fix #4886. Allow multiselection of session logs.

### CODE REVIEW

File: src/Ombu-Tests/OmDeferrerTest.class.st [hunk 1]
```diff
@@ -1,5 +1,5 @@
 "
-An EpDeferrerTest is a test class for testing the behavior of EpDeferrer
+An EpDeferrerTest is a test class for testing the behavior of OmDeferrer
```
Comment (tinchodias) [inline src/Ombu-Tests/OmDeferrerTest.class.st:2]: Offtopic: fix typo

**Comment by tinchodias** [comment]
By mistake, this branch includes the fix to #2853, which is not reviewed/merged yet. 

I think we can just wait to merge the other, without rebasing...

**Comment by tinchodias** [comment]
This PR is ready to be reviewed now that the fix to #2853 was merged.

**Review by MarcusDenker — APPROVED** [review]
tested the download from the CI, works

**Comment by tinchodias** [comment]
thanks Marcus

**Pull request merged**